pkg/diagrams/plantuml.go

```diff
@@ -16,7 +16,7 @@ import (
 
 const (
 	PlantUMLEnvVar  = "SYSL_PLANTUML"
-	PlantUMLDefault = "http://localhost:8080/plantuml"
+	PlantUMLDefault = "http://www.plantuml.com/plantuml" // "http://localhost:8080/plantuml"
```

Comment:
This is a dangerous default, since it will result in accidental leakage of work product from organisations using Sysl internally. A better solution is to keep the default but provide helpful instructions in the error output if localhost:8080 doesn't work.